### PR TITLE
:fire: added files to ignore while uploading

### PR DESCRIPTION
**Pull request by Archish27**

### CODE REVIEW

**Review by yashLadha — APPROVED** [review]

File: bin/helpers/constants.js [hunk 1]
```diff
@@ -131,7 +131,24 @@ const messageTypes = {
...
+const filesToIgnoreWhileUploading = [
+  '**/node_modules/**',
+  'node_modules/**',
+  'package-lock.json',
+  'package.json',
+  'browserstack-package.json',
+  'tests.zip',
+  'cypress.json',
+  '.idea/',
+  '*.iml',
```
Comment (karanshah-browserstack) [inline bin/helpers/constants.js:143]: Not needed as we only upload these allowedFileTypes fileTypes.

File: bin/helpers/constants.js [hunk 2]
```diff
@@ -131,7 +131,24 @@ const messageTypes = {
...
-const filesToIgnoreWhileUploading = ['**/node_modules/**', 'node_modules/**', 'package-lock.json', 'package.json', 'browserstack-package.json', 'tests.zip', 'cypress.json']
+const filesToIgnoreWhileUploading = [
+  '**/node_modules/**',
+  'node_modules/**',
+  'package-lock.json',
+  'package.json',
+  'browserstack-package.json',
+  'tests.zip',
+  'cypress.json',
+  '.idea/',
```
Comment (karanshah-browserstack) [inline bin/helpers/constants.js:142]: It is a regex and won't exclude subfolders in this. 
It should be `.idea/**`

**Review by karanshah-browserstack — CHANGES_REQUESTED** [review]

File: bin/helpers/constants.js [hunk 3]
```diff
@@ -131,7 +131,24 @@ const messageTypes = {
...
+  '*.iml',
+  '.vscode/',
+  '.npm/',
+  '.vscode-test',
+  '.yarn/cache',
```
Comment (karanshah-browserstack) [inline bin/helpers/constants.js:147]: Combine it with `.yarn/**`

**Review by karanshah-browserstack — CHANGES_REQUESTED** [review]

**Review by karanshah-browserstack — APPROVED** [review]
LGTM

File: bin/helpers/constants.js (hunk 3 above)
Comment (Archish27) [reply bin/helpers/constants.js]: fixed

**Pull request merged**